source/content/guides/object-cache-pro/02-installing-configuring.md

```diff
@@ -0,0 +1,316 @@
+
+1. Click the **Add** button under **Redis**. It might take a couple of minutes for the Object Cache server to come online. You will see a success message in a green box stating that Redis has been enabled.
+
+![Pantheon Settings Add-Ons](../../../images/guides/object-cache-pro/pantheon-settings-addons.png)
+
+### Activate Redis with Terminus
+
+Run the command below to enable Redis via the command line with Terminus:
+
+```bash{promptUser: user}
+terminus redis:enable <site>
+```
+
+## Installation and Configuration for Regular WordPress Sites
+
+1. Open the email you received after signing up for Early Access Object Cache Pro on Pantheon.
+
+	<Alert title="Note" type="info">
+
+	This link will always point to the latest version of Object Cache Pro. Be sure to store it in a safe place so you never miss out on updates to Object Cache Pro.
+
+	</Alert>
+
+1. Click the link in the email and download the file to install the Object Cache Pro plugin.
+
+1. Unzip the file and add it to your WordPress install as you normally would.
+
+	- There are some configuration options that you must add to set it up the plugin now that you have Object Cache Pro in your codebase. All of these options are stored in your `wp-config.php` in the root of your site repository.
+
+1. Add the license token in WordPress. Note that the license key will be provided by the platform in the future. Currently, you are responsible for adding it to your repository.
+
+1. Open the `wp-config.php` file and add the values below to the `WP_REDIS_CONFIG` constant somewhere above the `/* That's all, stop editing! Happy Pressing. */` line at the bottom of the file.
+
+		```php
+		/**
+		 * Object Cache Pro config
+		 */
+		define( 'WP_REDIS_CONFIG', [
+			'token' => '<LICENSE-TOKEN>',
+		] );
+		```
+
+1. Add Object Cache Pro configuration options after `define( 'WP_REDIS_CONFIG', [` in `wp-config.php` for a **standard WordPress** site. Note that there are more [configuration options](https://objectcache.pro/docs/configuration-options) that can be added. We recommend using the below settings which are optimized for Pantheon if you don't know specifically what each option does. The full, recommended contents of the `WP_REDIS_CONFIG` constant are:
+
+		```php
+		'token' => '<LICENSE-TOKEN>',
+		'host' => $_SERVER['CACHE_HOST'] ?? '127.0.0.1',
+		'port' => $_SERVER['CACHE_PORT'] ?? 6379,
+		'database' => $_SERVER['CACHE_DB'] ?? 0,
+		'password' => $_SERVER['CACHE_PASSWORD'] ?? null,
+		'maxttl' => 86400 * 7,
+		'timeout' => 1.0,
+		'read_timeout' => 1.0,
+		'debug' => false,
+		'analytics' => [
+			'enabled' => true,
+			'persist' => true,
+			'retention' => 3600, // 1 hour
+			'footnote' => true,
+			],
+		```
+
+1. Make sure you `git push` your changes up to your repository before you activate the plugin.
+
+1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin, locally with Lando, or via Terminus.
```

Comment:
@jazzsequence This step states that Lando can be used but no steps are provided. Should example steps for Lando be added or should the reference be removed?

Reply:
Can remove the reference to Lando.

Reply:
```suggestion
1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin, locally with WP-CLI, or via Terminus.
```

Reply:
```suggestion
1. Activate the Object Cache Pro plugin and enable Redis in the plugin from the WordPress Admin or via Terminus.
```

Reply:
Thanks @jazzsequence . I added WP-CLI.